### PR TITLE
Added few Reliance Retail brand stores

### PR DESCRIPTION
**Pull request by MG-LSJ**
Added
- Reliance Smart Point
- Reliance Smart Super Store
- Jio Store

### CODE REVIEW

File: data/brands/shop/supermarket.json [hunk 1]
```diff
@@ -30,6 +30,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "Reliance Smart Super Store",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Mall","Jio Mall","Jio Supermarket"],
+      "tags": {
+        "brand": "Reliance Smart Super Store",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/supermarket.json:38]: Is the brand "Reliance Smart" or "Reliance Smart Super Store"? https://relianceretail.com/reliance-smart.html would imply that it's just "Reliance Smart" (ignoring their choice of capitalization).

File: data/brands/shop/supermarket.json [hunk 2]
```diff
@@ -30,6 +30,17 @@
...
+        "brand:wikidata": "Q7311028",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/supermarket.json:39]: Probably should have a separate brand item instead of just putting everything under Reliance Retail.

File: data/brands/shop/convenience.json [hunk 3]
```diff
@@ -63,6 +63,17 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Smart Point",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Smart Point","Jio Smart","Jio Smart Point"],
+      "tags": {
+        "brand": "Reliance Smart Point",
+        "brand:wikidata": "Q7311028",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/convenience.json:72]: Probably should have a separate brand item instead of just putting everything under Reliance Retail.

File: data/brands/shop/supermarket.json (hunk 2 above)
Comment (MG-LSJ) [reply data/brands/shop/supermarket.json:39]: A Wikipedia page only exists for Reliance Retail where all these are mentioned as sub brands.

Comment (AntiCompositeNumber) [reply data/brands/shop/supermarket.json:39]:
```suggestion
        "brand:wikidata": "Q123565819",
```

File: data/brands/shop/convenience.json (hunk 3 above)
Comment (AntiCompositeNumber) [reply data/brands/shop/convenience.json:72]:
```suggestion
        "brand:wikidata": "Q123565801",
```

File: data/brands/shop/department_store.json [hunk 4]
```diff
@@ -12,6 +12,17 @@
...
+      "displayName": "Reliance Smart Bazar",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Mall","Jio Mall"],
+      "tags": {
+        "brand": "Reliance Smart",
+        "brand:wikidata": "Q7311028",
+        "name": "Reliance Smart Bazar",
+        "shop": "department_store"
+      }
+    },
```
Comment (AntiCompositeNumber) [inline data/brands/shop/department_store.json:25]: Looks like this is a hypermarket (combination supermarket and department store). As far as I can tell, Reliance aquired most of the previous Big Bazaar stores (but not all, and the deal to buy Big Bazaar's owner outright fell through) and rebranded them to Smart Bazaar. Big Bazaar is [in NSI](https://nsi.guide/index.html?t=brands&k=shop&v=supermarket&tt=big%20bazaar#bigbazaar-9ef8f9) as a supermarket, and it's my understanding that that's the preferred tagging for hypermarkets absent a more specific tag being created. 

https://www.wikidata.org/wiki/Q3631008 is the WD item for the former Big Bazaar. It (and the associated enwiki article) have been changed to Smart Bazaar, but I'm not sure if that's entirely correct.

tl;dr:

- Better as `shop=supermarket`
- Might replace Big Bazaar, if that brand has fully ceased operations (I can't quite tell).
- Unsure whether to use Q3631008 or a new WD item.

Comment (MG-LSJ) [reply data/brands/shop/department_store.json:25]: As per my local knowledge:

- **Smart Point** are small convenience stores (at least 10-20 locations dotted around in an avg Indian city)

- **Smart Bazaar** (rebranded from Big Bazaar) are usually multi story buildings (best suits `department_store` in osm wiki)

- **Smart Super Store** are Walmart equivalent  one large continuous store under the same roof
